src/Lecture1.hs

```diff
@@ -108,7 +116,14 @@ and finds a sum of the numbers inside this string.
 
 The string contains only spaces and/or numbers.
 -}
-strSum str = error "TODO"
+strSum :: String -> Int
+strSum str = strLstSum strlst 
+  where 
+    strlst = words str
+    strLstSum :: [String] -> Int
+    strLstSum sl = case sl of
+      [] -> 0
+      (x: xs) -> (read x :: Int) + strLstSum xs
```

Comment:
You can use standard functions to simplify the implementation of `strSum`:

* `words`: to split a string into a list of space-separated words
* `read`: to parse an `Int` from `String`
* `map`: to apply the given function to every single element of the list
* `sum`: to find a sum of all numbers in the list

If you use all the above standard functions, you'll be able to implement `strSum` as a one-line function 🚀